src/Algebra/Structures.agda

```diff
@@ -340,7 +340,7 @@ record IsNearSemiring (+ * : Op₂ A) (0# : A) : Set (a ⊔ ℓ) where
     }
 
   open IsMagma *-isMagma public
-    using ()
+    using (isEquivalence ; ∙-cong)
```

Comment:
- `isEquivalence` is already exported via `open IsMonoid +-isMonoid`?
- `∙-cong` is already exported via `*-cong`.

Please check if all the export changes are really needed!

Reply:
Ah good catch. Now I understand. Thanks @MatthewDaggitt  I will close this PR